aws_doc_sdk_examples_tools/metadata.py

```diff
@@ -205,12 +206,12 @@ class ExampleMergeMismatchedId(MetadataError):
 class Example:
     id: str
     file: str
+    languages: Dict[str, Language]
```

Comment:
Did these need to get reorganized?

Reply:
Yes, because you can't have args with defaults before args without.